### ISSUE TITLE
8.0.0475 build fails on Funtoo Linux: if_xcmdsrv.c:(.text+0x1efa): undefined reference to `sendToLocalVim'

### ISSUE THREAD

**Issue by fusion809** [vim/vim#1574]
Hi,

I tried compiling my [ebuilds](https://github.com/fusion809/vim-overlay/tree/master/app-editors) for v8.0.0475 but my vim and gvim ebuilds failed giving the error (with a little extra compile output to give you context):

```bash
OLD_PO_FILE_INPUT=yes msgfmt -v -o zh_CN.cp936.mo zh_CN.cp936.po
1634 translated messages, 3 fuzzy translations.
make[2]: Leaving directory '/var/tmp/portage/app-editors/gvim-8.0.0475/work/vim-8.0.0475/src/po'
make[2]: Entering directory '/var/tmp/portage/app-editors/gvim-8.0.0475/work/vim-8.0.0475/src/po'
make[2]: Nothing to be done for 'converted'.
make[2]: Leaving directory '/var/tmp/portage/app-editors/gvim-8.0.0475/work/vim-8.0.0475/src/po'
x86_64-pc-linux-gnu-gcc -c -I. -fwrapv  -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64  -I/usr/lib64/perl5/5.24.0/x86_64-linux/CORE  -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK  -I/usr/include/gtk-2.0 -I/usr/lib64/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -pthread -I/usr/include/pango-1.0  -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include     -mtune=generic -O2 -pipe -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        -o objects/if_perl.o auto/if_perl.c
x86_64-pc-linux-gnu-gcc -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK  -I/usr/include/gtk-2.0 -I/usr/lib64/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -pthread -I/usr/include/pango-1.0  -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include     -mtune=generic -O2 -pipe -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        version.c -o objects/version.o
link.sh: $LINK_AS_NEEDED set to 'yes': invoking linker directly.
  x86_64-pc-linux-gnu-gcc   -L. -Wl,-O1 -Wl,--sort-common -fstack-protector -rdynamic -Wl,-export-dynamic -Wl,--no-undefined -Wl,-E  -Wl,-O1 -Wl,--sort-common -L/usr/local/lib -Wl,--as-needed        -o gvim objects/arabic.o objects/buffer.o objects/blowfish.o objects/crypt.o objects/crypt_zip.o objects/dict.o objects/diff.o objects/digraph.o objects/edit.o objects/eval.o objects/evalfunc.o objects/ex_cmds.o objects/ex_cmds2.o objects/ex_docmd.o objects/ex_eval.o objects/ex_getln.o objects/farsi.o objects/fileio.o objects/fold.o objects/getchar.o objects/hardcopy.o objects/hashtab.o  objects/if_cscope.o objects/if_xcmdsrv.o objects/list.o objects/mark.o objects/memline.o objects/menu.o objects/misc1.o objects/misc2.o objects/move.o objects/mbyte.o objects/normal.o objects/ops.o objects/option.o objects/os_unix.o objects/pathdef.o objects/popupmnu.o objects/quickfix.o objects/regexp.o objects/screen.o objects/search.o objects/sha256.o objects/spell.o objects/spellfile.o objects/syntax.o objects/tag.o objects/term.o objects/ui.o objects/undo.o objects/userfunc.o objects/version.o objects/window.o objects/gui.o objects/gui_gtk.o objects/gui_gtk_x11.o objects/pty.o objects/gui_gtk_f.o objects/gui_beval.o objects/gui_gtk_gresources.o objects/if_lua.o  objects/if_perl.o objects/if_perlsfio.o objects/if_python.o objects/if_python3.o  objects/if_ruby.o   objects/netbeans.o objects/channel.o  objects/charset.o objects/json.o objects/main.o objects/memfile.o objects/message.o   -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype   -lSM -lICE -lXpm -lXt -lX11 -lXdmcp -lSM -lICE  -lm -lncurses -lelf -lnsl    -lacl -lattr -ldl  -L/usr/lib -llua -Wl,-E -Wl,-O1 -Wl,--sort-common -Wl,--as-needed  -L/usr/lib64/perl5/5.24.0/x86_64-linux/CORE -lperl -lpthread -lnsl -ldl -lm -lcrypt -lutil -lc    -lruby23 -lpthread -lgmp -ldl -lcrypt -lm  -L/usr/lib64    
objects/if_xcmdsrv.o: In function `serverSendToVim':
if_xcmdsrv.c:(.text+0x1efa): undefined reference to `sendToLocalVim'
collect2: error: ld returned 1 exit status
link.sh: Linking failed
Makefile:1902: recipe for target 'gvim' failed
make[1]: *** [gvim] Error 1
make[1]: Leaving directory '/var/tmp/portage/app-editors/gvim-8.0.0475/work/vim-8.0.0475/src'
Makefile:26: recipe for target 'first' failed
make: *** [first] Error 2
```

version 8.0.0474 build fine, so I know it is the latest patch that caused this error. 

Thanks for your time,
Brenton

**Comment by fusion809** [comment]
Testing to see if this is fixed by 8.0.0476...

**Comment by fusion809** [comment]
'tis fixed with 8.0.0476. Closing...
